.go-version

```diff
@@ -1 +1 @@
-1.70.9
+1.70.10
```

Comment:
ruh roh looks like someone [stole this version](https://github.com/lightstep/terraform-provider-lightstep/releases/tag/v1.70.10) and forgot to update

Reply:
this was me. I wasn't sure exactly what the release steps should be. I only merged PR and pushed up a lightweight tag, so if this is all that's missing perhaps we can drop a blurb in the README/DEVELOPING.md for the next passerby. Is there anything else? What's this file for if we don't need it to publish?

Reply:
yeah tbh i have no idea how this is _supposed_ to work... i think this file is just for bookkeeping so we don't have to remember which tags have been pushed already? but it would also be nice if merging a change to this file pushed a tag automatically.

it does feel like a quick update to the README is the easiest thing to do here, but itll still be super easy to forget this step